src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,44 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({
+  post: {
+    title,
+    user,
+    body,
+    comments,
+  },
+}) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {'Posted by '}
+        {user && (
+          <UserInfo user={user} />
+        )}
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {body}
+    </p>
+
+    <hr />
+
+    {!comments.length
```

Comment:
start from positive condition. Not critical in that case, but it is much more clear to read